assets/js/carousel.js

```diff
@@ -3,10 +3,30 @@ var flkty = new Flickity( elem, {
     // options
     cellAlign: 'left',
     contain: true,
-    autoPlay: 8000,
+    autoPlay: 4000,
     prevNextButtons: false,
     pageDots: false,
     draggable: false,
     wraparound: true,
     fade: true
-});
\ No newline at end of file
+});
+
+let carouselVideos = document.getElementsByClassName("carousel-video");
+let allCells = document.getElementsByClassName("carousel-cell");
+
+flkty.on( 'change', slideChange );
```

Comment:
Currently your are listening to the `change` event to pause all videos. Then listening to the `settle` event to resume all videos. This will not work for 2 reasons. 
1. While we should pause all hidden videos, we must only resume/play the video visible.
2. Settle fires after the animation has complete. Which means the video start too late. It should start as soon as visible.

So while the pausing all videos should wait for `settle` to make sure no video being fading away (still visible) is prematurely paused, the resuming should use `change` so the next video starts as soon as its fading in starts.

Something like:
```
flkty.on( 'change', function(index) {
   // Video matching `index` resume.
} );

flkty.on( 'change', function(index) {
   // All videos but the one matching `index` are paused.
} );
```

Let me know if I'm missing something.